### PR TITLE
Events calculator

### PR DESCRIPTION
**Pull request by Iptysam**
- Copy the files calculate.js and operate.js into a logic/ directory in your project.
- Analyze the files calculate.js and operate.js, be sure you understand what they do (note: you need to add [big.js](https://www.npmjs.com/package/big-js) as dependency).
- Import the files in your Calculator component.
- Implement the event handlers you need to use the math logic from calculate.js and operate.js in your React component.
- Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up.
- Test your app in the browser. It should be able to perform math operations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 > You will set up the environment and tools needed to develop a React application.
 > You will develop a React component that will hold the core functionality: a calculator.
+> Add the logic needed to make the Calculator component you developed in the previous project actually work
```
Comment (HAMMAS-SALEEM) [inline README.md:9]: ### Your Getting Started Section
Getting-Started2

- [ ] As you can see the image above. That's how your Getting Started section looks like. Kindly remove all the commands in the Getting Started Section and add the following commands in it 👍 to make your readme descriptive 😊

```
## Getting Started

`git clone git@github.com:Iptysam/Math-magicians.git`
`cd Math-magicians`
`npm start`

```

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Salam @Iptysam :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
:heavy_check_mark: Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Iptysam) [reply README.md:9]: Thank you

**Review by mikethreels — APPROVED** [review]
Hi Ipty, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**